cosmos/Parser/latex_parser/variable_extractor.py

```diff
@@ -28,14 +28,12 @@ def get_variables(latex):
     processed = processed.replace(')', '}')
 
     response = muterun_js(os.path.join(CURRENT_DIR, 'katex/parse.js'), processed)
-    # print(response.stderr)
     if response.stdout == b'-1\n' or not response.stdout:
         yield -1
         return
 
     tree = etree.fromstring(response.stdout)
     ast = tree.xpath('.//semantics')[0]
```

Comment:
Would prefer this area to be commented a little more and have descriptive variables. I'm not quite sure what's going on here.